app.js

```diff
@@ -47,6 +47,7 @@ app.use((err, req, res, next) => {
 })
 
 app.use((err, req, res, next) => {
+    console.log(err)
     res.status(500).send({msg: 'Internal Server Error'})
     next(err)
```

Comment:
This `500` error handling middleware should be the last one in the file, and so there should be no need for the `next(err)` because this is the end of the line for the error train 🚋